Marlin/enum.h

```diff
@@ -58,6 +58,22 @@ enum AxisEnum {
 #define LOOP_XYZE(VAR) LOOP_S_LE_N(VAR, X_AXIS, E_AXIS)
 #define LOOP_XYZE_N(VAR) LOOP_S_L_N(VAR, X_AXIS, XYZE_N)
 
+enum CalEnum {
+  _7P_STEP = 1,              // 7-point step - to change number of calibration points
+  _4P_STEP = _7P_STEP * 2,   // 4-point step
+  CEN      = 0,              // the 7 main calibration points - add definitions if needed
+  __A      = 1,
+  _AB      = __A + _7P_STEP,
+  __B      = _AB + _7P_STEP,
+  _BC      = __B + _7P_STEP,
+  __C      = _BC + _7P_STEP,
+  _CA      = __C + _7P_STEP,
+  NPP      = _7P_STEP * 6    // number of calibration points on the radius
+};
```

Comment:
It's not really the proper use of `enum`. I would recommend doing these as `constexpr uint8_t` instead.

Reply:
As these are only used by `G33`, they would be better defined just above the `G33` function in the .cpp file.

Reply:
Reworked this